plugins/modules/s3_bucket.py

```diff
@@ -120,6 +122,24 @@
     default: false
     type: bool
     version_added: 1.3.0
+  object_ownership:
+    description:
+      - Allow bucket's ownership controls.
+      - C(BucketOwnerPreferred) - Objects uploaded to the bucket change ownership to the bucket owner
+        if the objects are uploaded with the bucket-owner-full-control canned ACL.
+      - C(ObjectWriter) - The uploading account will own the object
+        if the object is uploaded with the bucket-owner-full-control canned ACL.
+      - This option cannot be used together with a I(delete_object_ownership) definition.
+    choices: [ 'BucketOwnerPreferred', 'ObjectWriter' ]
+    type: str
+    version_added: 1.5.0
```

Comment:
Nit; 1.5.0 was released last week. Could you please update the version here and in the option below to 2.0.0?